style.css

```diff
@@ -478,3 +483,45 @@ summary:focus {
   width: 16px;
   background-image: svg-load("./icon/button-right.svg");
 }
+
+menu.tabs {

```

Comment:
Incredible work 😍 

While you're still hacking on this, wanna rework these selectors to guarantee [proper aria attributes](https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html)?

That is:
* instead of `.tabs` we could do `[role=tablist]`
* instead of `.tabs li` we could do `[role=tab]`
* `[aria-selected]` instead of `li,active`

Happy to nitpick but just wanted to drop this in early :)

Reply:
Some exit criteria could be: does this render [reach-ui's Tabs](https://reacttraining.com/reach-ui/tabs/) without any extra effort? https://codesandbox.io/s/rough-tree-ghm6v 

We could then reference that in the docs :) Tabs is spec-compliant (and reviewed by some strong a11y talent!)

Reply:
Copy that. I'll change these for arias.

Reply:
> Some exit criteria could be: does this render [reach-ui's Tabs](https://reacttraining.com/reach-ui/tabs/) without any extra effort? https://codesandbox.io/s/rough-tree-ghm6v. Multiline could still be an extra class though.
> 
> We could then reference that in the docs :) Tabs is spec-compliant (and reviewed by some strong a11y talent!)

I do not know about this library. For testing i made mines in Vue like so
```HTML
<menu role="tablist">
  <li role="tab" v-for="(tab, key) in tabs" :key="key" :aria-selected="key == active"  @click="active = key">
    <a href="#" v-text="tab"></a>
  </li>
</menu>

<div class="window" role="tabpanel">
  <div class="window-body">
    <!-- some random lipsum content -->
  </div>
</div>

<script>
  const app = new Vue({
    el: '#app',
    data: {
      active: 0,
      tabs: ['Desktop', 'My computer', 'Control panel', 'Devices manager', 'Hardware profiles', 'Performance']
    }
  })
</script>
```

This is what renders what i captured above.